addon/components/frost-action-bar.js

```diff
@@ -153,14 +163,20 @@ export default Component.extend({
 
   // == Functions =============================================================
 
+  controlIsVisible (control) {
+    const isVisible = this.convertControl(control).isVisible
```

Comment:
I haven't walked through all the logic for this but `return !this.convertControl(control).isVisible` won't work?

Reply:
haha - leftovers from debugging. Ty.

Reply:
oh - and undefined needs to be handled like the prop is not set.